worker-delivery-validate.py

```
@@ -0,0 +1,127 @@
+    ver = doc["verification_commands"]
+    if not isinstance(ver, list) or not ver or any(not isinstance(i, dict) or not str(i.get("command", "")).strip() or i.get("exit") != 0 for i in ver):
+        fail("complete delivery requires verification_commands with exit 0")
+    if bound_cmd:
+        if bound_exit != "0": fail("bound verify command exited %s" % (bound_exit or "?"))
+        if not any(str(i.get("command", "")).strip() == bound_cmd and i.get("exit") == 0 for i in ver):
+            fail("report verification_commands do not match bound verify execution")
+    if doc["unresolved_failures"]: fail("unresolved failures remain: %s" % "; ".join(map(str, doc["unresolved_failures"])))
+    tracked = set(git(checkout, "ls-files").stdout.splitlines())
+    setup_cmds = [str(i.get("command", "")) for i in (doc.get("setup_commands") or []) if isinstance(i, dict) and i.get("exit") == 0]
+    missing = [name for name in LOCKS if name in tracked and not any(LOCK_INSTALL[name].search(cmd) for cmd in setup_cmds)]
+    if missing: fail("missing dependency install evidence for committed lockfiles: %s" % ", ".join(missing))
```

Ruff (0.16.3)

[error] Multiple statements on one line (colon)

(E701)